### DISCUSSION TITLE
Downloaded video seems to be corrupted

### DISCUSSION THREAD

**Discussion by MordenkainenDisjunction** [aclap-dev/video-downloadhelper#1510] [Help and Q&A]
### Your question

Hello,

I'm trying to download videos from https://justfor.fans/. The download seems to go pretty well but quite slow but when it finishes and I open the downloaded video it looks corrupted when opened in VLC player. See below image. A sample link to a video in the site would be https://api.xvid.com/v1/playlists/dash.mpd?file_id=64573828520eb22998cd642a,64573828520eb22998cd654f&autograph_tag=1161640-136.158.35.83&multi_use=true&redirect=true&expiry_time=1733158800&encryption=true&streaming=true&client_id=5a68e952e4b0de4febbd1a9a&signature=4bb7e651148d303f0e1ba375c91e479bee9767aa

See below my version information:
version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:133.0) Gecko/20100101 Firefox/133.0
{
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 13,
    "successfull_dl": 1
}

![image](https://github.com/user-attachments/assets/6da0628d-c436-4137-99dd-540c34e84d74)

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:133.0) Gecko/20100101 Firefox/133.0
{
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 13,
    "successfull_dl": 1
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I'm guessing here.  It looks like you are reporting a duplicate of this one: #1504.
